pipelines/pipelines/tensorflow/prediction/queries/ingest.sql

```diff
@@ -4,24 +4,31 @@
 -- you may not use this file except in compliance with the License.
 -- You may obtain a copy of the License at
 
---     https://www.apache.org/licenses/LICENSE-2.0
+--      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
is this change intentional?

Reply:
copied from another file I think - I will revert